pkgs/top-level/all-packages.nix

```
@@ -4881,7 +4881,11 @@ in
 
   parcellite = callPackage ../tools/misc/parcellite { };
 
-  patchutils = callPackage ../tools/text/patchutils { };
+  patchutils = patchutils_0_3_4;
```

```suggestion
  patchutils = callPackage ../tools/text/patchutils/0.3.4.nix { };
```
and can drop `patchutils_0_3_4` since we don't have a need to pin 0.3.4.